src/components/edit_notification.js

```diff
@@ -21,10 +21,10 @@ import MonascaClient from './monasca_client';
 export class EditNotificationPageCtrl {
 
   /** @ngInject */
-  constructor($scope, $injector, $location, backendSrv, datasourceSrv, alertSrv) {
+  constructor($scope, $injector, $location, backendSrv, datasourceSrv, alertSrv, monascaClientSrv) {
```

Comment:
Can we remove backendSrv, datasourceSrv now that you're injecting the monasca client? They don't appear to be used?
(Same for other instances)